listenbrainz/mbid_mapping/mapping/dump.py

```diff
@@ -0,0 +1,219 @@
+""" This module contains data dump creation and import functions.
```

Comment:
Adding this file here is fragile. 

The code inside `listenbrainz/mbid_mapping` runs in a separate docker image that only contains code inside `listenbrainz/mbid_mapping`. This is in contrast to all other containers which have all listenbrainz code in them. As a result, any import `listenbrainz` module inside `mbid_mapping` module will result in a `ModuleNotFoundError` at runtime.

While we know that this script is only run by dump_manager which is invoked through the normal listenbrainz containers, having this file here may give the wrong impression that it is runnable just inside the mbid_mapping container as well.  Further, if we import dump.py somewhere else inside the mbid_mapping module, it will probably to crash there.

Due to these reasons, I have been wondering for some time that `listenbrainz/mbid_mapping` should instead be a top level directory `mbid_mapping` to make this clearer.

Reply:
There's also issues that `listenbrainz/mbid_mapping` has a different `requirements.txt` file from rest of LB. So many deps we use elsewhere in codebase are unable there.

Reply:
moved to `listenbrainz/db/mapping_dump.py`